src/appkit.cs

```diff
@@ -5943,6 +6150,16 @@ interface NSDrawerDelegate {
 
 	}
 
+	[Protocol]
+	interface NSFontChanging
+	{
+		[Export ("changeFont:")]
+		void ChangeFont ([NullAllowed] NSFontManager sender);
+
+		[Export ("validModesForFontPanel:")]
+		NSFontPanelModeMask ValidModesForFontPanel (NSFontPanel fontPanel);
```

Comment:
is the name already present somewhere ?
if not it would be better to have a verb/action and remove the prefix, e.g. `GetValidModes`